app/Login.tsx

```diff
@@ -46,13 +47,18 @@ export function Login(props: { onLoginSuccess: () => void }): ReactElement {
             display: "flex",
             flexDirection: "column",
             flexGrow: 4,
-            color: "#FFFFFF",
             fontSize: "12px",
           }}
         >
           <Form.Label>Username</Form.Label>
+          <div style={{ height: "10px" }}></div>
           <Form.Control
-            style={{ backgroundColor: "#B5A8A8" }}
+            style={{
+              width: "160px",
+              height: "32px",
+              flexShrink: "0",
+              background: "#B5A8A8",
+            }}
```

Comment:
this stuff seems like it was in the other PR as well, but doesn't make sense for me to review it twice (that's what i meant earlier by whether they're completely independent)